flake.lock

```diff
@@ -0,0 +1,412 @@
+{
+  "nodes": {
+    "alejandra": {
+      "inputs": {
+        "fenix": "fenix",
+        "flakeCompat": "flakeCompat",
+        "nixpkgs": [
+          "dream2nix",
+          "nixpkgs"
```

Comment:
I am getting some changes in this file when I run `nix build`, like:

```git
-        "nixpkgs": [
-          "dream2nix",
-          "nixpkgs"
-        ]
+        "nixpkgs": "nixpkgs_2"
```

Could be because we are running different versions of nix?

Reply:
Yes, since the version of Nix you're running is very old, it doesn't support the `default` output attribute of a flake. Get the latest version of Nix like `nix shell github:nixos/nix#nix` and then try again.